libraries/AC_AutoTune/AC_AutoTune_Multi.cpp

```diff
@@ -908,36 +912,23 @@ void AC_AutoTune_Multi::set_tuning_gains_with_backoff(AxisType test_axis)
     case TuneType::RATE_D_UP:
         break;
     case TuneType::RATE_D_DOWN:
-        switch (test_axis) {
-        case AxisType::ROLL:
-            tune_roll_rd = MAX(min_d, tune_roll_rd * AUTOTUNE_RD_BACKOFF);
-            tune_roll_rp = MAX(AUTOTUNE_RP_MIN, tune_roll_rp * AUTOTUNE_RD_BACKOFF);
-            break;
-        case AxisType::PITCH:
-            tune_pitch_rd = MAX(min_d, tune_pitch_rd * AUTOTUNE_RD_BACKOFF);
-            tune_pitch_rp = MAX(AUTOTUNE_RP_MIN, tune_pitch_rp * AUTOTUNE_RD_BACKOFF);
-            break;
-        case AxisType::YAW:
-            tune_yaw_rLPF = MAX(AUTOTUNE_RLPF_MIN, tune_yaw_rLPF * AUTOTUNE_RD_BACKOFF);
-            tune_yaw_rp = MAX(AUTOTUNE_RP_MIN, tune_yaw_rp * AUTOTUNE_RD_BACKOFF);
-            break;
-        case AxisType::YAW_D:
-            tune_yaw_rd = MAX(min_d, tune_yaw_rd * AUTOTUNE_RD_BACKOFF);
-            tune_yaw_rp = MAX(AUTOTUNE_RP_MIN, tune_yaw_rp * AUTOTUNE_RD_BACKOFF);
-            break;
-        }
         break;
     case TuneType::RATE_P_UP:
         switch (test_axis) {
         case AxisType::ROLL:
-            tune_roll_rp = MAX(AUTOTUNE_RP_MIN, tune_roll_rp * AUTOTUNE_RP_BACKOFF);
+            tune_roll_rd = tune_roll_rd * (1.0f - gain_backoff);
+            tune_roll_rp = tune_roll_rp * (1.0f - gain_backoff);
             break;
         case AxisType::PITCH:
-            tune_pitch_rp = MAX(AUTOTUNE_RP_MIN, tune_pitch_rp * AUTOTUNE_RP_BACKOFF);
+            tune_pitch_rd = tune_pitch_rd * (1.0f - gain_backoff);
```

Comment:
Don't you still want to use AUTOTUNE_RP_MIN?

Reply:
Checked during the tuning process.